chipsec/modules/common/ia32cfg.py

```diff
@@ -43,23 +43,24 @@ def check_ia32feature_control(self):
         self.logger.start_test( "IA32 Feature Control Lock" )
         self.logger.log( "[*] Verifying IA32_Feature_Control MSR is locked on all logical CPUs.." )
 
-        ok = True
+        ok = ModuleResult.PASSED
         for tid in range(self.cs.msr.get_cpu_thread_count()):
-            #feature_cntl = chipsec.chipset.read_register( self.cs, 'IA32_FEATURE_CONTROL', tid )
-            #chipsec.chipset.print_register( self.cs, 'IA32_FEATURE_CONTROL', feature_cntl )
-            feature_cntl_lock = self.cs.get_control('Ia32FeatureControlLock' )
-            self.logger.log( "[*] cpu%d: IA32_Feature_Control Lock = %d" % (tid,feature_cntl_lock) )
-            if 0 == feature_cntl_lock: ok = False
+            if self.logger.VERBOSE:
+                feature_cntl = self.cs.read_register( 'IA32_FEATURE_CONTROL', tid )
+                self.cs.print_register( self.cs, 'IA32_FEATURE_CONTROL', feature_cntl )
+            feature_cntl_lock = self.cs.get_control('Ia32FeatureControlLock', tid )
+            self.logger.log( "[*] cpu{:d}: IA32_Feature_Control Lock = {:d}".format(tid,feature_cntl_lock) )
+            if 0 == feature_cntl_lock:
+                ok = ModuleResult.FAILED
```

Comment:
We should rename the variable 'ok' as this implies a Boolean value.  Something like result would be better now that we are passing ModuleResult around in it.